Cargo.toml

```diff
@@ -26,9 +26,9 @@ alloy = { git = "https://github.com/alloy-rs/alloy", features = [
 
 # zk-evm dependencies
 plonky2 = "0.2.2"
-evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-trace_decoder = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-proof_gen = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
+evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm.git", branch = "develop" }
```

Comment:
It's probably worth waiting until @Nashtare does the release for this

Reply:
I already have the new versions ready. I can re-run the entire Ethereum test suite + John's 200 Shanghai L1 blocks that we saved to check for regression and we could do the release if it's pressing.

Reply:
Sounds sensible.

Reply:
The Ethereum test suite passed fine on the `zk_evm@prep_release` branch, as well as all John's L1 block witnesses (apart from two that OOMed, retrying them as I write this). So we should be good with rolling the new versions soon.